packages/react-native/ReactAndroid/src/test/java/com/facebook/react/modules/network/HeaderUtilTest.kt

```diff
@@ -24,26 +23,26 @@ class HeaderUtilTest {
 
   @Test
   fun nameStripKeepsLetters() {
-    assertEquals(ALPHABET_TEST, HeaderUtil.stripHeaderName(ALPHABET_TEST))
+    assertThat(HeaderUtil.stripHeaderName(ALPHABET_TEST)).isEqualTo(ALPHABET_TEST);
   }
 
   @Test
   fun nameStripKeepsNumbers() {
-    assertEquals(NUMBERS_TEST, HeaderUtil.stripHeaderName(NUMBERS_TEST))
+    assertThat(HeaderUtil.stripHeaderName(NUMBERS_TEST)).isEqualTo(NUMBERS_TEST);
   }
 
   @Test
   fun nameStripKeepsSpecials() {
-    assertEquals(SPECIALS_TEST, HeaderUtil.stripHeaderName(SPECIALS_TEST))
+    assertThat(HeaderUtil.stripHeaderName(SPECIALS_TEST)).isEqualTo(SPECIALS_TEST);
   }
 
   @Test
   fun nameStripDeletesTabs() {
-    assertEquals(TABULATION_STRIP_EXPECTED, HeaderUtil.stripHeaderName(TABULATION_TEST))
+    assertThat(HeaderUtil.stripHeaderName(TABULATION_TEST)).isEqualTo(TABULATION_STRIP_EXPECTED);
   }
 
   @Test
   fun nameStripRemovesExtraSymbols() {
-    assertEquals(BANNED_TEST_EXPECTED, HeaderUtil.stripHeaderName(NAME_BANNED_SYMBOLS_TEST))
+    assertThat(HeaderUtil.stripHeaderName(VALUE_BANNED_SYMBOLS_TEST)).isEqualTo(VALUE_BANNED_SYMBOLS_TEST);
```

Comment:
This is wrong and is causing tests to fail:

```

> Task :packages:react-native:ReactAndroid:testDebugUnitTest

HeaderUtilTest > nameStripRemovesExtraSymbols FAILED
    org.junit.ComparisonFailure at HeaderUtilTest.kt:46
```

Reply:
@cortinico, adjusted!